bentoml/__init__.py

```diff
@@ -1,3 +1,19 @@
+"""
```

Comment:
This is helpful in notebook and python shell:

```
import bentoml
help(bentoml)
```
<img width="385" alt="Screen Shot 2022-07-08 at 5 30 27 AM" src="https://user-images.githubusercontent.com/489344/177992521-95166b75-df64-4ef7-a8f8-ea98e0b4bab9.png">